### PR TITLE
qa: do the set/get attribute on the remote filesystem

### PR DESCRIPTION
**Pull request by joscollin**
mount_b wasn't mounted with the remote filesystem.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [x] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Comment by joscollin** [comment]
@vshankar You've reviewed this code earlier. https://github.com/ceph/ceph/commit/be551a6c68018207b6c730ae9d12341c796f5deb. But I think it should be this way to set/get attribute on a remotefs. WDYT?

**Comment by joscollin** [comment]
jenkins test windows

**Comment by joscollin** [comment]
jenkins test make check

**Comment by joscollin** [comment]
jenkins test make check arm64

File: qa/tasks/cephfs/test_mirroring.py [hunk 1]
```diff
@@ -1494,8 +1494,21 @@ def test_get_set_mirror_dirty_snap_id(self):
         """
         That get/set ceph.mirror.dirty_snap_id attribute succeeds in a remote filesystem.
         """
+        log.debug('reconfigure client auth caps')
```
Comment (vshankar) [inline qa/tasks/cephfs/test_mirroring.py:1497]: While this change is fine, the test is just validating the xattr set/get which is equally good to be done on the primary fs. But I agree, the test comment should match up with the test case :)

**Review by vshankar — APPROVED** [review]

File: qa/tasks/cephfs/test_mirroring.py (hunk 1 above)
Comment (joscollin) [reply qa/tasks/cephfs/test_mirroring.py:1497]: ok, So what do you suggest? Change the test comment or this change is fine?

Comment (vshankar) [reply qa/tasks/cephfs/test_mirroring.py:1497]: I approved the PR -- LGTM.

**Comment by joscollin** [comment]
jenkins test make check arm64

**Comment by vshankar** [comment]
This PR is under test in [https://tracker.ceph.com/issues/67711](https://tracker.ceph.com/issues/67711).

**Comment by joscollin** [comment]
jenkins test make check arm64

**Comment by joscollin** [comment]
jenkins test make check arm64

**Review by vshankar — APPROVED** [review]
https://tracker.ceph.com/projects/cephfs/wiki/Main#wip-vshankar-testing-20240826122843-debug

**Pull request merged**